lbrynet/analytics/api.py

```diff
@@ -3,32 +3,24 @@
 import logging
 
 from requests import auth
-from requests_futures import sessions
+from txrequests import Session
 
 from lbrynet import conf
 from lbrynet.analytics import utils
 
-
 log = logging.getLogger(__name__)
 
 
 def log_response(fn):
-    def _log(future):
-        if future.cancelled():
-            log.warning('Request was unexpectedly cancelled')
-        elif future.exception():
-            exc, traceback = future.exception_info()
-            log.warning('Failed to send an analytics event', exc_info=(type(exc), exc, traceback))
```

Comment:
I think exc_info is necessary to get the real traceback and not a useless trace from twisted internals